src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -301,6 +302,23 @@ def setup_config(self) -> None:
             if self.token:
                 inmanta_config.Config.set(section, "token", self.token)
 
+        # Create a raw config object, with only the part of the configuration that will be
+        # common for the local and remote project compiles (environment and authentication)
+        raw_config = {
+            "config": {"environment": str(self.environment)},
+            "compiler_rest_transport": {},
+            "client_rest_transport": {},
+        }
+        if self.token:
+            raw_config["compiler_rest_transport"]["token"] = self.token
+            raw_config["client_rest_transport"]["token"] = self.token
+
+        # Persist environment and token info in the inmanta config file of the project
+        # to make sure it is sent to the remote orchestrator
+        project_path = pathlib.Path(self.local_project._path)
+        config_file_path = project_path / ".inmanta"
+        config_file_path.write_text(toml.dumps(raw_config))
```

Comment:
Why/ when is this required? i.e. the server should set this up correctly for the compiler when the server runs the compiler?

Reply:
It failed when running the export service entities plugin we run during the sync of the project, I guess because this compile is not triggered by the orchestrator itself?

Reply:
We could try to trigger the export via the api instead of doing it manually, that might be a cleaner approach, but more work to implement, I would then defer it to another ticket